grading/load_config_json.cpp

```diff
@@ -95,11 +95,20 @@ void AddAutogradingConfiguration(nlohmann::json &whole_config) {
         // Handle single file or array of files.
         if (exe_name->is_array()) {
           for (typename nlohmann::json::iterator itr = exe_name->begin(); itr != exe_name->end(); itr++) {
-            whole_config["autograding"]["compilation_to_runner"].push_back(*itr);
+            if (itr->is_string()) {
+              std::string name = to_string(*itr);
```

Comment:
`to_string` is not the best choice here. According to the [docs](https://json.nlohmann.me/api/basic_json/to_string/), `to_string` is used for getting the serialized result which is why you have to strip the quotes. I think you should just be able to get the value from the iterator. Or maybe std's to_string could help. There is definitely a way to accomplish this without stripping quotes.